### ISSUE TITLE
Installation documentation seems to be missing

### ISSUE THREAD

**Issue by alex4200** [lightbody/browsermob-proxy#727]
I want to try to use 'browsermob' to monior the network connections during a selenium test, but it seems this project missed an installation documentation. 

Example:
- How to install 'browsermob-proxy'?
- How to start the proxy server? Where to find that executable?

**Comment by mediga** [comment]
This document here gives the information you need. Let me know if you are looking for any specific steps.

https://github.com/lightbody/browsermob-proxy

There are two modes to start browsermob

1) Embedded mode . install it by configuring maven dependency. Refer to "Getting started: Embedded Mode" in the above documentation

2) Standalone mode. Refer to "Getting started: Standalone" section in the above documentation

**Comment by alex4200** [comment]
Here are some points I think are missing or are misleading: 

- A general description of what this toolactually is (A proxy that runs to which selenim must connect, or something like that). 
- In the section 'Embedded Mode': I have no idea what a 'pom' is...
- In the section 'Embedded Mode': Example only provided for java. But I am using the python implementation of selenium. 
- In the section 'Getting started: Standalone': It would be nice to have known that 'browsermob-proxy' is a file that is part of that repository, and you have to clone it, and where to find that file.
- On the documentation page https://browsermob-proxy-py.readthedocs.io/en/stable/ there is a description on how to use this tool with python, but it does not seem to work as I get errors, and it is not clear WHAT to use as the path. 
- In addition, complete installation instructions are missing. When trying to use this tool in python I do 'from browsermobproxy import Server', which will fail of course. I need to 'git clone' the repository in order to be able to download the code to a place I can easily access (to access the script ''browsermob-proxy), and in addition, I have to 'pip install' the same tool. 
- Also undlear how to use this tool in general with python-selenium. Just run the script as explained in the example_ Or do I have to start the proxy server before and provide the second port number...? nclear to me ... 
- Details on how to use the proxy on a mac are missing.
- The 'browsermob-proxy' script actually does not make sense, as I was not able to find any 'jar' files in the repository
- Then there are downloadable files available here: https://github.com/lightbody/browsermob-proxy/releases. So now I have THREE installation. Are they all the same? Why don't any of them contain a jar file required for that actual proxy?

**Comment by thibaut-sticky** [comment]
@alex4200 You talk about the Python version. This last is maintained by other guys: https://github.com/AutomatedTester/browsermob-proxy-py

So for sure you complain on the wrong repo.

**Comment by alex4200** [comment]
@thibaut-sticky : Yes, you are partly right. Its not just this repo, but also the python repo. 

But still, some pieces of the documentation for browsermob-proxy could be improved. I am offering my help on that...
